avaframe/ana5Utils/DFAPathGeneration.py

```diff
@@ -436,21 +385,172 @@ def extendProfileBottom(cfg, dem, profile):
     vDirZ = np.sum(vDirZ)
     vDirX, vDirY, vDirZ = DFAtls.normalize(np.array([vDirX]), np.array([vDirY]), np.array([vDirZ]))
     # extend in this direction
-    gamma = cfg.getfloat('factBottomExt') * sLast / np.sqrt(vDirX**2 + vDirY**2)
+    factExt = cfg.getfloat('factBottomExt')
+    gamma = factExt * sLast / np.sqrt(vDirX**2 + vDirY**2)
     xExtBottom = np.array([xLast + gamma * vDirX])
     yExtBottom = np.array([yLast + gamma * vDirY])
-    # project on cszDEM
-    zExtBottom, _ =  gT.projectOnGrid(xExtBottom, yExtBottom, zRaster, csz=csz)
+    # project on DEM
+    zExtBottom, _ = gT.projectOnGrid(xExtBottom, yExtBottom, zRaster, csz=csz)
+    # ToDo: If this point is not on the dem dicothomie to find the last point on the extention and on the dem
```

Comment:
Not sure what is meant by this sentence...
Should this TODO be addressed right away?